catroid/build.gradle

```
@@ -185,9 +185,9 @@ android {
         targetSdkVersion 26
         applicationId appId
         testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
-        versionCode 63
+        versionCode 64
```

```suggestion
        versionCode 65
```

Necessary because we have uploaded 64 already once.

---

Done